test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -1157,6 +1193,49 @@ static void ipsec_test_capability(void)
 	CU_ASSERT(odp_ipsec_capability(&capa) == 0);
 }
 
+static void test_ipsec_stats(void)
+{
+	ipsec_test_flags flags = {0};
+	struct cipher_param *cipher;
+	struct auth_param *auth;
+	uint32_t a, c;
+	int rc = -1;
+
+	for (c = 0; c < ARRAY_SIZE(ciphers); c++)
+		for (a = 0; a < ARRAY_SIZE(auths); a++) {
+			rc = test_esp_out_in(&ciphers[c], &auths[a], &flags);
+			if (rc == 0)
+				goto stats_tests;
+		}
+
+	for (c = 0; c < ARRAY_SIZE(cipher_auth_comb); c++) {
+		rc = test_esp_out_in(&cipher_auth_comb[c].cipher,
+				     &cipher_auth_comb[c].auth,
+				     &flags);
+		if (rc == 0)
+			goto stats_tests;
+	}
+
+stats_tests:
+	CU_ASSERT_EQUAL_FATAL(rc, 0);
```

Comment:
I do not think it is a fatal error if there are no supported algorithms. I suppose that is the case with null cypto, i.e. when not compiling linux-gen with openssl.

Reply:
This is actually removed in the latest version (since we are running for all algos).